pkg/api/alertmanager.go

```diff
@@ -93,26 +93,29 @@ import (
 // swagger:parameters RouteCreateSilence
 type CreateSilenceParams struct {
 	// in:body
-	Body SilenceBody
+	Silence PostableSilence
 }
 
-//swagger:parameters RouteGetSilence RouteDeleteSilence
+// swagger:parameters RouteGetSilence RouteDeleteSilence
 type GetDeleteSilenceParams struct {
 	// in:path
 	SilenceId string
 }
 
-// swagger:model
-type SilenceBody struct {
-	Id string `json:"id"`
-	amv2.Silence
+// swagger:parameters RouteGetSilences
+type GetSilencesParams struct {
+	// in:query
+	Filter []string
 }
 
 // swagger:model
-type GettableSilences amv2.GettableSilences
+type PostableSilence = amv2.PostableSilence
+
+// swagger:model
+type GettableSilences = amv2.GettableSilences
```

Comment:
From a brief research, I was unsure whenever there would be any downsides of using type aliasing vs struct embedding. In this case, I don't want to add any extra methods so I went with the former. Happy to listed to opinions.